compiler+runtime/src/cpp/jank/util/cli.cpp

```diff
@@ -25,6 +25,7 @@ namespace jank::util::cli
                    opts.profiler_file,
                    "The file to write profile entries (will be overwritten).");
     cli.add_flag("--gc-incremental", opts.gc_incremental, "Enable incremental GC collection.");
+    cli.add_flag("--fdirect-call", opts.direct_calls, "Compile with direct calls.");
```

Comment:
Please use `--direct-call`. Also, let's provide a more helpful message. The flag already says "direct call", but what does that mean? For example "Elide var derefs when compiling function calls."